blog/index.jade

```diff
@@ -0,0 +1,18 @@
+h1 Blog
+p Some stuff we want to say about our blog.
+
+//- show blog teaser roll
+for post, slug in public.blog.posts._data
+  if post.layout !== false
+
+    article.h-entry
+      h1.p-name: a(href="/blog/posts/#{ slug }")= post.title
+      p
+        | Published by&nbsp;
+        a.p-author.h-card(href='http://example.com') #{ post.authors }
```

Comment:
Since we have multiple authors we need iterate over each one if we are going to link their names. 

Also, we need to decide what the link should be: their twitter, github or website? Maybe best not to link each author? What do you think?